index.html

```diff
@@ -83,5 +83,270 @@ <h1 class="text-dark hero-title">
         </button>
       </div>
     </section>
+
+    <section class="work-section">
+      <header>
+        <h1 class="section-header text-center">My Recent Works</h1>
+        <span class="underline"></span>
+      </header>
+
```

Comment:
- [ ] Why are you using the `header` tag here :smile: please remember that the `header` typically contains the logo and the navigations so it would be better if you consider using `div` :+1: